themes/src/main/resources/theme/keycloak/login/resources/css/tile.css

```diff
@@ -166,6 +166,10 @@
     --pf-c-tile__icon--FontSize: var(--pf-c-tile--m-display-lg__header--m-stacked__icon--FontSize);
 }
 
```

Comment:
There's an issue with overflow context when the title is too long. Is it possible to add `overflow:hidden` property to `.pf-c-tile` class? UI is broken on mobile devices then.

![long](https://user-images.githubusercontent.com/38039883/112652599-dc9f8580-8e4d-11eb-9bcb-5ce8ecb60305.png)